### PR TITLE
Add job/pull-ci-openshift-release-ci-secret-bootstrap-config-validation

### PR DESCRIPTION
**Pull request by hongkailiu**
To avoid mistake like https://github.com/openshift/release/pull/16084/files#diff-6f809450f5216bc90d0c08b723c9fe080da1358283bbf47c42f05bfc589c49fdR1757

/cc @openshift/openshift-team-developer-productivity-platform

### CODE REVIEW

**Review by petr-muller — APPROVED** [review]

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml [hunk 1]
```diff
@@ -1,5 +1,31 @@
 presubmits:
   openshift/release:
+  - agent: kubernetes
+    always_run: false
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:4]: Huh why? Does this take this much time?

**Review by petr-muller — APPROVED** [review]

**Comment by petr-muller** [comment]
/lgtm cancel
accidently approved

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml (hunk 1 above)
Comment (hongkailiu) [reply ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml]: I thought `always_run` must be false if `run_if_changed` is set.

So `always_run` means for every PR it runs.
https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md

Updated.

Comment (hongkailiu) [reply ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml]: Ha. I was right.

```yaml
{"component":"unset","error":"error loading prow config: job pull-ci-openshift-release-ci-secret-bootstrap-config-validation is set to always run but also declares run_if_changed targets, which are mutually exclusive","file":"prow/cmd/checkconfig/main.go:210","func":"main.main","level":"fatal","msg":"Validation failed","severity":"fatal","time":"2021-02-25T13:22:26Z"}
```

Comment (hongkailiu) [reply ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml]: Removed `run_if_changed` as well.
I just feel we do not need to run it if the config is not changed.

Comment (petr-muller) [reply ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml]: Heh, I wouldn't bother if the thing runs fast /shrug

**Review by petr-muller — APPROVED** [review]

**Pull request merged**